app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
```

Comment:
you have three different Errors and one message for them. And the logic question, do you need 3 Errors?)

Reply:
How it's can be? If message the same from task.
![Screenshot (87)](https://user-images.githubusercontent.com/110389820/194514306-7dc492cd-cd0a-468d-93bf-f4e5221040a9.png)
![Screenshot (86)](https://user-images.githubusercontent.com/110389820/194514308-73725139-de2c-48d9-b06c-613c5a7d4b89.png)

Reply:
yes, but create different messages for these errors